runtime/port/sysvipc/j9shmem.c

```diff
@@ -1348,10 +1348,14 @@ j9shmem_createDir(struct J9PortLibrary* portLibrary, char* cacheDirName, uintptr
 		Trc_PRT_j9shmem_createDir_ExitFailedFile();
 		break;
 	case EsIsDir:
-#if !defined(J9ZOS390)
-		if (J9SH_DIRPERM_ABSENT == cacheDirPerm) {
+#if !defined(J9ZOS390) && !defined(OPENJ9_BUILD)
+		if (J9SH_DIRPERM_ABSENT == cacheDirPerm || J9SH_DIRPERM_ABSENT_GROUPACCESS == cacheDirPerm) {
 			/* cacheDirPerm option is not specified. Change permission to J9SH_DIRPERM. */
```

Comment:
comment needs to be updated to match the new code

Reply:
Fixed